packages/jest-cli/src/TestRunner.js

```diff
@@ -123,43 +127,9 @@ class TestRunner {
 
   runTests(testPaths: Array<string>, maybeReporter?: TestReporter) {
```

Comment:
hm... i really don't understand what `maybeReporter` is and i'm confused why eslint didn't catch that.
I'll try to figure out how to sync it